tensorflow/contrib/tensorrt/test/biasadd_matmul_test.py

```diff
@@ -144,14 +144,6 @@ def ShouldRunTest(self, run_params):
     # mode, which is a bug. Re-enable this when trt library is fixed.
     return not trt_test.IsQuantizationMode(run_params.precision_mode)
 
-  def ExpectedAbsoluteTolerance(self, run_params):
```

Comment:
Is this supposed to be deleted?

Reply:
Yes because we updated the test base class to use smaller atol/rtol, and after we fixed the python3 issues in order to make the test pass we need to use (inherit) those atol/rtol, that's why we're deleting this.